omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java

```diff
@@ -50,7 +49,6 @@ public void controller(FragmentConfiguration config,
 						patient = patientService.getPatient(NumberUtils.toInt(patientId));
 					}
 				}
-			}
```

Comment:
![image](https://user-images.githubusercontent.com/9057882/92948644-f9e4a780-f459-11ea-90ec-1a67b80e72a0.png)
Don't you have one tab too many in the marked column?

Reply:
You removed one if nesting, so the entire block should move one tab to the left.

Reply:
new changes made